src/views/settings/SettingsSubjects.tsx

```diff
@@ -146,6 +146,8 @@ const SettingsSubjects: Screen<"SettingsSubjects"> = ({ navigation }) => {
     });
   }, [navigation, colors.primary]);
 
+  const [customColor, setCustomColor] = useState("");
```

Comment:
Pour éviter que <TextInput> soit vide 
```suggestion
  const [customColor, setCustomColor] = useState(selectedSubject[1].color);
```
Et si ça fonctionne, il faudrait peut-être remplacer `selectedSubject[1].color` => `customColor` pour une automatisation du choix des couleurs